compiler/src/dotty/tools/dotc/transform/PatternMatcher.scala

```diff
@@ -265,8 +265,13 @@ object PatternMatcher {
       def unapplySeqPlan(getResult: Symbol, args: List[Tree]): Plan = args.lastOption match {
         case Some(VarArgPattern(arg)) =>
           val matchRemaining =
-            if (args.length == 1)
-              patternPlan(getResult, arg, onSuccess)
+            if (args.length == 1) {
+              val toSeq = ref(getResult)
+                .select(defn.Seq_toSeq.matchingMember(getResult.info))
```

Comment:
Why not simply: `.select(nme.toSeq)`?

Reply:
The code is defensive here to support possible overloading of `toSeq`.

Reply:
We should add tests for this then. Would it work to do `.select(nme.toSeq).appliedToNone()`?

Reply:
My hunch is that it will crash the compiler, as `appliedToNone` will not do overloading resolution.

Reply:
But there is no need right? You cannot have two overloads of a method that takes no parameter

Reply:
Never mind, your solution seems better. `matchingMember` is already used in multiple places in this file (I suppose for reasons)